cypress/e2e/donor/add-correspondent.cy.js

```diff
@@ -84,4 +84,27 @@ describe('Add correspondent', () => {
                 });
             });
     });
+
+    it.only('warns when name shared with donor', () => {
```

Comment:
```suggestion
    it('warns when name shared with donor', () => {
```

Reply:
Interesting, the stop-only pre-commit hook didn't pick this up 🤔